app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -78,6 +118,75 @@ describe('components/ConnectedSidebar', () => {
         </MemoryRouter>
       </Provider>
     );
+
+    const submenu = wrapper
+      .find('SubMenu[title="Admin"]')
+      .first()
+      .last()
+      .prop('children') as ReactWrapper[];
+
+    expect(submenu[3].key).toMatch('location');
+  });
+
+  it('displays menu links for enabled Form Configuration module', () => {
```

Comment:
@ashfahan Do we need to add this test case?. It was catered for in `Test order of menu` test case

Reply:
removed in favor of `Test order of menu`